### PR TITLE
chore: update meta image

### PR DESCRIPTION
**Pull request by boojack**
Add domain prefix for header meta image so that it can show the featured image when sharing in twitter.
image

### CODE REVIEW

**Comment by boojack** [comment]
FYI @milasuperstar

File: pages/blog/_slug/index.vue [hunk 1]
```diff
@@ -83,6 +83,10 @@ export default {
   },
   head() {
     const blog = (this as any).blog;
+    let featureImage = blog.featureImage;
+    if (startsWith(featureImage, "/")) {
+      featureImage = "https://www.bytebase.com" + featureImage;
```
Comment (tianzhou) [inline pages/blog/_slug/index.vue:88]: how about adding hostname to env https://sourcegraph.com/github.com/bytebase/bytebase.com/-/blob/nuxt.config.js?L212:5

Comment (boojack) [reply pages/blog/_slug/index.vue]: Updated

**Comment by boojack** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**